astropy_timeseries/sampled.py

```diff
@@ -161,22 +171,38 @@ def from_pandas(self, df):
         # TODO: determine how user can specify time scale
         time = Time(df.index)
 
-        # Create table without the time column
-        table = Table.from_pandas(df)
+        if LooseVersion(astropy.__version__) > LooseVersion("3.1.1"):
```

Comment:
I think this should be <3.2 instead of >3.1.1 since the functionality is not going to be present in the 3.1 branch.

Reply:
Could you define ``ASTROPY_LT_32`` at the top of the file and then use that constant here?